mod-invoice-storage/schemas/export_configuration.json

```diff
@@ -30,7 +30,7 @@
     "uploadURI": {
       "description": "URI batch vouchers should be uploaded to.  Limited to FTP initially",
       "type": "string",
-      "pattern": "^ftp://([\\.a-z0-9]+)(/([\\./a-z0-9]?)+)?$"
+      "pattern": "^ftp://([\\.\\_\\-a-z0-9]+)(/([\\./\\-\\_\\.a-z0-9]?)+)?$"
```

Comment:
URI from example do not pass this regex,

Reply:
🤔 They does for me... how are you testing it?

Reply:
```
package cmcnally.sandbox;

import java.util.regex.Pattern;

public class App {
  public static void main(String[] args) {

    Pattern p = Pattern.compile("^ftp://([\\.\\_\\-a-z0-9]+)(/([\\./\\-\\_\\.a-z0-9]?)+)?$");
    System.out.println(p.matcher("ftp://ftp.amherst-lib.edu/invoices/").matches());
  }
}
```
returns true for me

Reply:
🤔it works for me as well. I must have missed something last time.